sql-server-2014-alwayson-existing-vnet-and-ad/azuredeploy.json

```diff
@@ -0,0 +1,531 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+
+    "newStorageAccountNamePrefix": {
+      "type": "string",
+      "maxLength": 18,
+      "metadata": {
+        "description": "Naming prefix for each new storage account created. Three storage accounts will be created using this string as a prefix for the name. 18-char max, lowercase alpha"
+      },
+      "defaultValue": "xxxsql"
+    },
+
+    "storageAccountType": {
+      "type": "string",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_GRS",
+        "Standard_RAGRS",
+        "Premium_LRS"
+      ],
+      "metadata": {
+        "description": "Type of new Storage Accounts (Standard_LRS, Standard_GRS, Standard_RAGRS or Premium_LRS) to be created to store VM disks"
+      },
+      "defaultValue": "Premium_LRS"
+    },
+
+    "vmNamePrefix": {
+      "type": "string",
+      "maxLength": 8,
+      "metadata": {
+        "description": "Naming prefix for each VM name. 8-char max, lowercase alpha"
+      },
+      "defaultValue": "xxxsql"
+    },
+
+    "sqlVMSize": {
+      "type": "string",
+      "allowedValues": [
+        "Standard_D1",
+        "Standard_DS1",
+        "Standard_D2",
+        "Standard_DS2",
+        "Standard_D3",
+        "Standard_DS3",
+        "Standard_D4",
+        "Standard_DS4",
+        "Standard_D11",
+        "Standard_DS11",
+        "Standard_D12",
+        "Standard_DS12",
+        "Standard_D13",
+        "Standard_DS13",
+        "Standard_D14",
+        "Standard_DS14"
+      ],
+      "metadata": {
+        "description": "Size of the SQL VM instances to be created"
+      },
+      "defaultValue": "Standard_DS3"
```

Comment:
Default to non premium sizes just to be cost effective for new users deploying this ?